inc/c_util/tarray_ll.h

```diff
@@ -99,6 +105,51 @@ TARRAY_LL(T) TARRAY_LL_CREATE(C)(void)
     return result;                                                                                                          \
 }
 
+/*introduces a function definition for tarray_create_with_capacity*/
+#define TARRAY_LL_CREATE_WITH_CAPACITY_DEFINE(C, T)                                                                         \
+TARRAY_LL(T) TARRAY_LL_CREATE_WITH_CAPACITY(C)(uint32_t capacity)                                                           \
```

Comment:
>TARRAY_LL_CREATE_WITH_CAPACITY [](http://example.com/codeflow?start=13&length=30)

can TARRAY_LL_CREATE be then replaced by a default call to TARRAY_LL_CREATE_WITH_CAPACITY where capacity is "1"?

Less code...

Reply:
Sure.